pagarme/pagarme-magento
# credit card i18n: csv file

### Descrição

Removes unused PagarMe_CreditCard.csv and keeps the
PagarMe_Creditcard.csv.

### Testes Realizados

Manual test:
 - Check the cc form labels, when the store locale is Português (Brasil) (system -> general -> general -> locale options -> locale)
<!Descreva aqui todos os detalhes realizados para assegurar o Pull Request. Coloque todos os dados possíveis: versões dos recursos, módulos extras adicionados ao teste :)>

<!NÃO SE ESQUEÇA DE: Marcar um dos desenvolvedores do Pagar.me para review.>
